src/dinerl_util.erl

```diff
@@ -0,0 +1,66 @@
+-module(dinerl_util).
```

Comment:
You might want to consider implementing [`erl_stat`](https://github.com/SemanticSugar/erl_utils/blob/master/src/erl_stat.erl) here.

Reply:
Or, at least, using its functions instead of reimplementing them.

Reply:
That was exactly my initial intention but then I realized this is public and erl_stat is not open-sourced just yet :(

Reply:
Oh, right! 🤦‍♂️

Reply:
Maybe it's worth just bringing `erl_stat` to this project and using it. That way, if we ever make `erl_utils` public, we can just remove the copied module from here and everything keeps working as usual.

Reply:
That is a really good suggestion. Ill keep it in mind for the next iteration that is soon to come. Merging to get this deployed.